european-journal-of-human-genetics.csl

```diff
@@ -15,7 +15,7 @@
     <issn>1018-4813</issn>
     <eissn>1476-5438</eissn>
     <summary>Superscript Vancouver style for some NPG journals, with et-al-min="7" and et-al-use-first="3"</summary>
```

Comment:
The summary was already correct, but the style itself wasn't. That's ... very peculiar.